syntax = "proto3";

package example.v1;

import "google/protobuf/any.proto";
import "google/protobuf/struct.proto";
import "google/protobuf/timestamp.proto";
import "google/protobuf/wrappers.proto";

message Repeated {
  repeated int64 the_int = 1;
  repeated Wat the_enums = 2;
  repeated Thing the_msg = 3;
  repeated google.protobuf.Timestamp the_timestamps = 4;
  repeated google.protobuf.Any the_anys = 5;
  repeated google.protobuf.StringValue the_strs = 6;
  repeated google.protobuf.Struct the_structs = 7;
}

enum Wat {
  WAT_UNSPECIFIED = 0;
  WAT_WAT = 1;
}

message Thing {
  int64 wat = 2;
}